test/e2e/tests/connections/edit-networks-flow.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { withFixtures, WINDOW_TITLES } from '../../helpers';
+import { DEFAULT_FIXTURE_ACCOUNT, DAPP_HOST_ADDRESS } from '../../constants';
+import FixtureBuilder from '../../fixture-builder';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import Homepage from '../../page-objects/pages/home/homepage';
+import TestDapp from '../../page-objects/pages/test-dapp';
+import PermissionListPage from '../../page-objects/pages/permission/permission-list-page';
+import SitePermissionPage from '../../page-objects/pages/permission/site-permission-page';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+
+describe('Edit Networks Flow', function () {
```

Comment:
nit: for the same reason as above

```suggestion
describe('Edit Networks Permissions', function () {
```

Reply:
Done, thanks.